addons/crm/tests/test_crm_lead.py

```diff
@@ -758,3 +758,58 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertEqual(lead.mobile, self.test_phone_data[2])
         self.assertFalse(lead.phone_sanitized)
+
+    @users('user_sales_manager')
+    def test_crm_lead_alias_help(self):
+        crm_team_model = self.env['ir.model']._get_id('crm.team')
+        crm_lead_model = self.env['ir.model']._get_id('crm.lead')
+
+        # archive other teams
+        self.env['crm.team'].search([]).active = False
+
+        user_team_leads, team_leads, user_team_opport, team_opport = self.env['crm.team'].create([{
+            'name': 'UserTeamLeads',
+            'use_leads': True,
+            'member_ids': [(6, 0, [self.env.user.id])],
+            'company_id': self.env.company.id,
+        }, {
+            'name': 'TeamLeads',
+            'use_leads': True,
+            'member_ids': [],
+            'company_id': self.env.company.id,
+        }, {
+            'name': 'UserTeamOpportunities',
+            'use_leads': False,
+            'member_ids': [(6, 0, [self.env.user.id])],
+            'company_id': self.env.company.id,
+        }, {
+            'name': 'TeamOpportunities',
+            'use_leads': False,
+            'member_ids': [],
+            'company_id': self.env.company.id,
+        }])
+
+        self.env['crm.lead'].create([{
+            'name': 'LeadOurTeam',
+            'team_id': user_team_leads.id,
+            'type': 'lead',
+        }, {
+            'name': 'LeadTeam',
+            'team_id': team_leads.id,
+            'type': 'lead',
+        }, {
+            'name': 'OpportunityOurTeam',
+            'team_id': user_team_opport.id,
+            'type': 'opportunity',
+        }, {
+            'name': 'OpportunityTeam',
+            'team_id': team_opport.id,
+            'type': 'opportunity',
+        }])
+
+        # archive our team one by one and check that we have the correct help message
+        crm_lead = self.env['crm.lead']
+        for team in [user_team_leads, team_leads, user_team_opport, team_opport]:
+            team.alias_id.sudo().write({'alias_name': team.name})
+            self.assertIn(team.alias_id.display_name, crm_lead.get_empty_list_help(""))
```

Comment:
Maybe check the whole url while we're at it? `<a href='mailto:%s'>%s</a>`, makes it more obvious we're comparing to html too.